packages/firebase/index.d.ts

```diff
@@ -108,7 +108,7 @@ declare namespace firebase {
      *     resolve. This does not apply if the user is anonymous.</dd>
      * </dl>
      *
-     * @return {!firebase.Promise<void>}
+     * @return {!Promise<void>}
```

Comment:
Please remove all @return that doesn't have any comment since it can be inferred from the type definition itself.

Reply:
Done